docs/guide/tables.md

```diff
@@ -0,0 +1,125 @@
+# Tables
+
+Since the SIX Web Components Library supports basic components only, we do not provide any table
+implementation. If your application depends on tables, you should consider using a 3rd party table
+implementation like [AG Grid](https://www.ag-grid.com/).
+
+To fit the SIX style and make the tables look more like a SIX Web Component we suggest you to
```

Comment:
Proposal: 

"Overwriting AG Grid its standard theme allows you to present the table according to the SIX corporate styleguide."